### ISSUE TITLE
make Hitting a chair with a cable creates a noose

### ISSUE THREAD

**Issue by Lightning38088** [yogstation13/yogstation#2030]
Hippie has this mechanic and it's pretty fun (for some reason) to do and even escape out of, best of all i used it once to hang my target and make it look like they commited suicide. Would be a fun little meme mechanic.

**Comment by 00aquir** [comment]
THERE'S NO CEILING IN 2D SPESSMAN, IT'S A LIE!

**Comment by Jc1469** [comment]
:thinking: 

did you know if you have a phone and you stand on a stool and suicide you hang yourself

**Comment by Lightning38088** [comment]
yeah but who the fuck has the time to go get it if they aint a engineer.

**Comment by Mrcreepssss** [comment]
... there's cables everywhere in maint. and art storage, which has zero acces restriction and filled with cables. i don't think that would be an issue.

**Comment by McDonald072** [comment]
Lightning meant getting the phone, not cables
